services/web/server/tests/unit/with_dbs/02/test_projects_cancellations.py

```diff
@@ -102,6 +103,46 @@ async def test_copying_large_project_and_aborting_correctly_removes_new_project(
             slow_storage_subsystem_mock.delete_project.assert_called_once()
 
 
+@pytest.mark.parametrize(*standard_user_role_response())
+async def test_copying_large_project_and_retrieving_copy_task(
+    client: TestClient,
+    logged_user: dict[str, Any],
+    primary_group: dict[str, str],
+    standard_groups: list[dict[str, str]],
+    user_project: dict[str, Any],
+    expected: ExpectedResponse,
+    catalog_subsystem_mock: Callable,
+    slow_storage_subsystem_mock: MockedStorageSubsystem,
+    project_db_cleaner: None,
+):
+    assert client.app
+    catalog_subsystem_mock([user_project])
+
+    # initiate a project copy that will last long (simulated by a long running storage)
+    # POST /v0/projects
+    create_url = client.app.router["create_projects"].url_for()
+    assert str(create_url) == f"{API_PREFIX}/projects"
+    create_url = create_url.with_query(from_study=user_project["uuid"])
+    resp = await client.post(f"{create_url}", json={})
+    data, error = await assert_status(resp, expected.accepted)
+    assert not error
+    assert data
+    assert "task_id" in data
```

Comment:
why not validate against response model? kind of like ``parse_obj_as(TaskGet, data)`` ?

Reply:
nah. this is not yet so implemented. I'm keeping this for later.